src/app/video-alumno/video-alumno.component.ts

```diff
@@ -109,31 +115,42 @@ export class VideoAlumnoComponent {
   }
 
   getContentMark() {
-    this.retroalimentacionService.getMarcasXacontenido(parseInt(this.idContent, 10)).subscribe(
-      (val: any) => {
-        this.marcas = val.results;
-        console.log('POST call successful value returned in body',
-          val);
-      },
-      response => {
-        console.log('POST call in error', response);
-      },
-      () => {
-        console.log('The POST observable is now completed.');
-      });
+    this.retroalimentacionService
+      .getMarcasXacontenido(parseInt(this.idContent, 10))
+      .subscribe(
+        (val: any) => {
+          this.marcas = val.results;
+          console.log("POST call successful value returned in body", val);
+        },
+        response => {
+          console.log("POST call in error", response);
+        },
+        () => {
+          console.log("The POST observable is now completed.");
+        }
+      );
   }
 
   getContentInteractiveDetail(idContent) {
     if (idContent !== undefined) {
-      this.contenidoService.getDetalleContenidoInteractivo(idContent).subscribe(contenido => {
-        this.contenidoInt = contenido;
-        this.id = contenido.contenido.url.split('watch?v=')[1];
-        this.contentsLoaded = Promise.resolve(true);
-        console.log('contenido alumno', contenido);
-        console.log('idd', this.id);
-      }, error => {
-        console.log('Error getting question information -> ', error);
-      });
+      this.contenidoService.getDetalleContenidoInteractivo(idContent).subscribe(
+        contenido => {
+          //True si el video es solo lineal y no se puede saltar entre marcas, false de lo contrario
+          contenido.linear = true;
+          this.isVideoLinear = contenido.linear;
+          this.contenidoInt = contenido;
+          this.id = contenido.contenido.url.split("watch?v=")[1];
+          this.contentsLoaded = Promise.resolve(true);
+          console.log("contenido alumno", contenido);
+          console.log("idd", this.id);
+          if (this.isVideoLinear) {
```

Comment:
Misma pregunta, creo que no deberíamos tener 2 variables: `this.isVideoLinear` y `canJump`. Me parece mejor dejar solo una

Reply:
Esto es para deshabilitar el input de la barra de progreso, que no se había tenido en cuenta